src/api/posts/package.json

```diff
@@ -25,6 +28,8 @@
     "normalize-url": "6.1.0"
   },
   "devDependencies": {
-    "supertest": "6.1.6"
+    "@senecacdot/eslint-config-telescope": "1.0.0",
+    "supertest": "6.1.6",
+    "eslint": "7.32.0"
```

Comment:
These should be listed in alphabetical order

```
"@senecacdot/eslint-config-telescope": "1.0.0",
    "eslint": "7.32.0",
    "supertest": "6.1.6"
```

Reply:
Do I need to fix back the order ? ```
"@senecacdot/eslint-config-telescope": "1.0.0",
"eslint": "7.32.0",
 "supertest": "6.1.6" ```

Reply:
Because it's a `@` ?

Reply:
Yes please